docs: add jcloud version doc (#4929)
jina-ai/jina: docs/fundamentals/jcloud/advanced.md

```{figure} external-executors-multiple.png
:width: 70%
```
## Deploy with specific `jina` version

> When deploying Flow to `jcloud`, the default `jina` version would be used. If for any reason you'd like your Flow to be built with a specific `jina` version, you can do so by using `version` arg while deploying the Flow.

Reviewer: I'd avoid talking about the default jina version on jcloud, as it depends on when we can merge the dependabot PR.

```suggestion
To manage `jina` version while deploying a Flow to `jcloud`, you can pass `version` arg in the Flow yaml.
```

Author: makes sense
